guidelines/MONAI-Workloads.md

```diff
@@ -0,0 +1,1149 @@
+
+**Clinical Workflow**
+
+
+
+1. Studies get routed in a large hospital DICOM routing system, at peak hour, the hospital is seeing about 300 studies per hour.
+2. As series arrive to DICOM router, DICOM router sends selected slices out of the series an AI processing system
+3. The AI processing system selects the correct slice or two for classification based on DICOM metadata.
+4. The AI algorithm evaluates the image metadata with the pixels and determines if the metadata (i.e. left shoulder) is indeed what the pixel data is showing (classification for left shoulder from image data).
+5. After classifying the image the AI system sends a response back to the appropriate DICOM target location.
+6. After receiving the response the DICOM router can route the image accordingly, either to the rejected queue for further analysis by QA technicians, or if everything looks good to the radiology work queue.
+
+**Clinical Workflow (without DICOM Router)**
+
+
+
+1. Studies are initiated on a modality and the DICOM is sent directly to a generalized application that ingests raw DICOM files.
+2. The application will read the DICOM metadata and select a single image or slice from the series and send a significantly reduced dataset to the AI platform and algorithm specific for the QA study associated with the dataset. Note: in this example the reduction is in some aspect of the overall data set to simplify the AI processing complexity in proportion to the algorithm accuracy.   In some variations, the application will send several image metadata and pixel sets in parallel to an AI platform and parse the results collectively.
+3. The AI platform will evaluate the image metadata and pixels to validate the pair. In this example a classification solution is used and the results are sent back to the application.
+4. After classification results are sent back to the application it adds metadata to the DICOM images that can be used later to filter the correct series of images for radiology studies. These DICOMs are added to the PACs along with the original data and made available to QA technicians.
+5. Optionally and in addition to the previous action, the application can directly send corrected DICOM studies or augmented DICOMs to additional AI processing pipelines with different algorithms. 
+
+**Clinical Use Case: **Study Metadata QA
+
+**Computational Workload: **[Synchronous](#synchronous-computational-workload)
+
+**Clinical workload:** 400 studies per hour, 1,200 series per hour, 1 slice per series, response time requirement of &lt; 500 milliseconds per slice.
+
+**Importance to Clinical AI Deployment: **critical, this type of inference is expected to be the most common across imaging, as it can drive a lot of other use cases downstream 
+
+
+
+
+**Figure:** Study Metadata QA without DICOM Router
+
+![MONAI_Metadata Quality Assurance](assets/MONAI_Metadata_Quality_Assurance.jpg)
```

Comment:
The description is in the use case steps listed before each figure.  I can add to each Clinical Use case a line that states
This scenario is described below in detail in Figure MONAI Metadata Quality Assurance.  
Otherwise, it is implied that each figure titled the same as the Clinical Use case describes at least one version of the use case.

Reply:
I'm adding the following sentence in this section that applies to all Figures in the Clinical Use case
 _Each figure in this section is a sequence diagram described by the use case steps in the same section._